cirq_superstaq/job_test.py

```diff
@@ -31,8 +32,13 @@ def new_job() -> css.Job:
     return css.Job(client, "my_id")
 
 
-def mocked_get_job_requests(*job_dicts: Dict[str, Any]) -> mock._patch:
-    """Mocks the server's response to `get_job` requests using the given sequence of job_dicts."""
+def mocked_get_job_requests(
+    *job_dicts: Dict[str, Any]
+) -> "mock._patch[Union[MagicMock, AsyncMock]]":
```

Comment:
nit
```suggestion
) -> "mock._patch[Union[mock.MagicMock, mock.AsyncMock]]":
```
(we use "mock.MagicMock" elsewhere in this file so it feels weird to import it separately for this line)

Reply:
also fwiw - not sure which is more correct in this case, but it seems to pass type check for me with either
```suggestion
) -> "mock._patch[mock.MagicMock]":
```
or
```suggestion
) -> "mock._patch[mock.Mock]":
```

Reply:
Will do the last suggestion.